css-text-3/Overview.bs

```diff
@@ -1670,7 +1670,9 @@ Text Processing</h3>
              Note: [=Hanging=] the white space rather than collapsing it
              allows users to see the space when selecting or editing text.
           <li>If 'white-space' is set to ''break-spaces'',
-             [=hanging=] or collapsing the advance width of the spaces or tabs
+             [=hanging=] or collapsing the advance width of the [=white space=],
+	     [=preserved=] [=tabs=],
+	     or [=other space separators=]
```

Comment:
I think spaces -> white space here wasn't necessary given we're calling out tabs explicitly?

Reply:
This should probably be rephrased, but what I'm trying to do ties into what I was saying in https://github.com/w3c/csswg-drafts/pull/4269/files#r320544043.
This includes:
* preserved spaces (regardless of whether they started started their life as U+0020, U+0009, or a line feed)
* preserved tabs
* other space separators

but does not include preserved new lines.

So if we do the redefining of `preserved white space` proposed in https://github.com/w3c/csswg-drafts/pull/4269/files#r320544043, this would become
> [...] the advance width of the preserved white space, preserved tabs, or other space separators [...]